src/klipse/klipse_editors.cljs

```diff
@@ -166,7 +168,9 @@
                    (compactize-map {:idle-msec idle-msec
                                     :on-completion (when (= "clojure" editor-in-mode)
                                                      #(trigger-autocomplete in-editor (j/call-in js/window [:klipse_clj :lang :clojure :completions] (current-token in-editor))))
-                                    :on-should-eval #(eval-in-codemirror-editor eval-fn result-element in-editor snippet-args editor-out-mode state)}))
+                                    :on-should-eval #((go
+                                                        (>! on-eval-channel :reload))
+                                                      (eval-in-codemirror-editor eval-fn result-element in-editor snippet-args editor-out-mode state))}))
```

Comment:
fix indentation. This form should be vertically aligned with the `go` form

Reply:
The `eval-in-codemirror-editor` call is outside of `go` block, which only runs `(>! on-eval-channel :reload)`.